lms/urls.py

```diff
@@ -1019,3 +1012,12 @@
             name='submit_financial_assistance_request'
         )
     )
+
+urlpatterns += (
```

Comment:
We should move this to `openedx/stanford/djangoapps/sneakpeak/urls.py` and include it from there.